tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -169,10 +171,10 @@ PCM_PLAYBACK_ADD(Headphone, 0, PIPELINE_PCM_1)
 PCM_CAPTURE_ADD(Headset mic, 1, PIPELINE_PCM_2)
 PCM_PLAYBACK_ADD(SDW1-speakers, 2, PIPELINE_PCM_3)
 ifdef(`MONO', `', `PCM_PLAYBACK_ADD(SDW2-speakers, 3, PIPELINE_PCM_4)')
-PCM_CAPTURE_ADD(DMIC, 4, PIPELINE_PCM_5)
-PCM_PLAYBACK_ADD(HDMI1, 5, PIPELINE_PCM_6)
-PCM_PLAYBACK_ADD(HDMI2, 6, PIPELINE_PCM_7)
-PCM_PLAYBACK_ADD(HDMI3, 7, PIPELINE_PCM_8)
+PCM_CAPTURE_ADD(DMIC, DMIC_ID, PIPELINE_PCM_5)
+PCM_PLAYBACK_ADD(HDMI1, eval(DMIC_ID + 1), PIPELINE_PCM_6)
+PCM_PLAYBACK_ADD(HDMI2, eval(DMIC_ID + 2), PIPELINE_PCM_7)
+PCM_PLAYBACK_ADD(HDMI3, eval(DMIC_ID + 3), PIPELINE_PCM_8)
```

Comment:
what kernel PR changes this? We should merge only when both kernel and topology are ready

Reply:
@RanderWang please link this PR to your corresponding kernel PR.